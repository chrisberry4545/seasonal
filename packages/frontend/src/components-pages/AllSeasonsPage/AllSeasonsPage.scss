@import './../../../node_modules/@chrisb-dev/seasonal-shared-frontend-components/dist/styles/settings';
@import './../../../node_modules/@chrisb-dev/seasonal-shared-frontend-components/dist/styles/tools';

.c-all-seasons-page {
  display: flex;
  flex-direction: column;
  width: 100%;
  @include mq($from: tablet) {
    width: calc(100% - #{$sidebar-width-desktop});
  }
}
