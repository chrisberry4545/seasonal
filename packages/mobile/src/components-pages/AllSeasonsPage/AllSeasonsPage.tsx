import {
  AllSeasonsFoodConnector,
  AllSeasonsRecipesConnector
} from '../../components-main';
import { CreateFoodAndRecipeTabs } from '../../components-layout';

export const AllSeasonsPage = CreateFoodAndRecipeTabs({
  foodScreen: AllSeasonsFoodConnector,
  recipeScreen: AllSeasonsRecipesConnector
});
