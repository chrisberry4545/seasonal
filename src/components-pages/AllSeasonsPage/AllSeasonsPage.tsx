import React, { SFC } from 'react';
import {
  AllSeasonsConnector,
  HeaderConnecter
} from '../../components-main';
import { MainContainer } from '../../components-layout';

export const AllSeasonsPage: SFC<{}> = () => (
  <MainContainer>
    <HeaderConnecter />
    <AllSeasonsConnector />
  </MainContainer>
);
