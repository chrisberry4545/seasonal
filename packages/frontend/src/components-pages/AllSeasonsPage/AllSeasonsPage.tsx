import React, { FC } from 'react';
import {
  PageWithMenu,
  AllSeasonsFoodConnector,
  AllSeasonsGraphConnector,
  AllSeasonsRecipesConnector,
  BottomTabsConnecter
} from '../../components-main';

import './AllSeasonsPage.scss';

export const AllSeasonsPage: FC<{}> = () => (
  <PageWithMenu>
    <div className='c-all-seasons-page'>
      <AllSeasonsGraphConnector />
      <AllSeasonsFoodConnector />
      <AllSeasonsRecipesConnector />
      <BottomTabsConnecter />
    </div>
  </PageWithMenu>
);
