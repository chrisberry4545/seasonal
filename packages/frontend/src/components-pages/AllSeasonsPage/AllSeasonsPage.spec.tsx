import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { AllSeasonsPage } from '../AllSeasonsPage/AllSeasonsPage';

jest.mock('../../components-main', () => ({
  AllSeasonsConnector: () => 'AllSeasonsConnector',
  AllSeasonsGraphConnector: () => 'AllSeasonsGraphConnector',
  PageWithMenu: () => 'PageWithMenu'
}));

describe('<AllSeasonsPage />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() =>
    wrapper = shallow(
      <AllSeasonsPage />
    )
  );

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());

});
