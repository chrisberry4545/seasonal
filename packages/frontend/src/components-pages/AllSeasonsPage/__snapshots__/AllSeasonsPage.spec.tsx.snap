// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AllSeasonsPage /> renders correctly 1`] = `
<PageWithMenu>
  <div
    className="c-all-seasons-page"
  >
    <AllSeasonsGraphConnector />
    <AllSeasonsConnector />
  </div>
</PageWithMenu>
`;
